bootstrap/ansible/playbook.yml

```diff
@@ -1,284 +1,491 @@
+
+        - name: Create Python virtual environment directory
+          file:
+            path: "{{ git_prefix }}/venv"
+            state: directory
+            mode: 0755
+            owner: "{{ djangooperator }}"
+            group: apache
+
+        - name: Initialize Python virtual environment
+          shell: >
+              test -f {{ git_prefix }}/venv/bin/activate
+              || (python3.6 -m venv {{ git_prefix }}/venv
+              && echo created)
+          register: venv_register
+          become_user: "{{ djangooperator }}"
+          changed_when: venv_register.stdout == "created"
+
+        # Install Django application dependencies, so that psycopg2, which is
+        # required to create the PostgreSQL database and user, is installed.
+
+        - name: Install Django application dependencies
+          pip:
+            requirements: "{{ git_prefix }}/{{ reponame }}/requirements.txt"
+            executable: "{{ git_prefix }}/venv/bin/pip3"
+          become_user: "{{ djangooperator }}"
+
+        # Create a PostgreSQL database and a user for the Django application.
+
+        - name: Create a new PostgreSQL database
+          postgresql_db:
+            name: "{{ db_name }}"
+          become_user: postgres
+          vars:
+            ansible_python_interpreter: "{{ git_prefix }}/venv/bin/python3"
+
+        - name: Create a PostgreSQL admin user and grant permissions
+          postgresql_user:
+            name: "{{ db_admin_user }}"
+            db: "{{ db_name }}"
+            priv: ALL
+            password: "{{ db_admin_passwd }}"
+            role_attr_flags: CREATEDB
+          become_user: postgres
+          vars:
+            ansible_python_interpreter: "{{ git_prefix }}/venv/bin/python3"
+
+        # Configure Apache.
+
+        - name: Check if the mod_wsgi module from Python 3.6 is installed to Apache
+          stat:
+            path: /etc/httpd/conf.modules.d/02-wsgi.conf
+          register: mod_wsgi_installed
+
+        - name: Install the mod_wsgi module from Python 3.6 to Apache
+          shell: "{{ git_prefix }}/venv/bin/mod_wsgi-express install-module --modules-directory /etc/httpd/modules > /etc/httpd/conf.modules.d/02-wsgi.conf"
+          when: not mod_wsgi_installed.stat.exists
+
+        - name: Create a WSGI configuration file from a template if SSL is disabled
+          template:
+            src: "{{ git_prefix }}/{{ reponame }}/bootstrap/ansible/wsgi_conf.tmpl"
+            dest: /etc/httpd/conf.d/{{ wsgi_conf_file_name }}
+          when: not ssl_enabled
+
+        - name: Create a WSGI configuration file from a template if SSL is enabled
+          template:
+            src: "{{ git_prefix }}/{{ reponame }}/bootstrap/ansible/wsgi_conf_ssl.tmpl"
+            dest: /etc/httpd/conf.d/{{ wsgi_conf_file_name }}
+          when: ssl_enabled
+
+        # Configure the Django application.
+
+        - name: Copy the ColdFront local_settings.py file from the sample
```

Comment:
Settings templates change very often, so these should be refreshed each time. (Everything including and below the WSGI configuration file generation should be moved to `common`.)